### ISSUE TITLE
I cannot use any of the functionalities advertised for Vale Server

### ISSUE THREAD

**Issue by politician** [errata-ai/vale-vscode#69]
# How to reproduce

## Prerequisite

Install Vale, Vale Server, Vale VS Code extension:

```sh
brew install vale
brew install --cask vale-server
code --install-extension errata-ai.vale-server
```

Start Vale server:

```sh
open -a "Vale Server"
```

Create a project based on the [most basic configuration](https://vale.sh/generator/):

```sh
mkdir ~/test-vale
cd ~/test-vale

cat <<EOF >> .vale.ini
StylesPath = styles

MinAlertLevel = suggestion
Vocab = Base

[*]
BasedOnStyles = Vale
EOF

cat <<EOF >> README.md
This is an undknown word.
EOF

vale sync
```

## Error 1

Open the `README.md` in VS Code.

The output for the Vale extensions:

```sh
E100 [--config] Runtime error

path '/Users/romain/Library/Application Support/errata-ai/Vale Server/_vale.ini' does not exist

Execution stopped with code 1.
```

Try to switch between the _Server_, _Compatibility_, modes, the error does not change.

However, with the _Command line_ mode, the extension works (without the server-specific features of course).

## Error 2

Add a config file to the path mentioned in the error:

```sh
cat <<EOF > "$HOME/Library/Application Support/errata-ai/Vale Server/_vale.ini"
StylesPath = "$HOME/test-vale/styles"

MinAlertLevel = suggestion
Vocab = Base

[*]
BasedOnStyles = Vale
EOF
```

The extension now works in all three modes. However, the quick fixes and vocab management aren't available in any of the three modes.

![Screenshot 2022-06-20 at 22 55 31](https://user-images.githubusercontent.com/3155568/174640008-b8b41f68-9b9c-4a80-b31c-6d4dcefffc9b.png)

![Screenshot 2022-06-20 at 22 56 54](https://user-images.githubusercontent.com/3155568/174640260-c8c6aa66-bbe8-4d21-8b01-4bddb1870e83.png)

![Screenshot 2022-06-20 at 23 09 21](https://user-images.githubusercontent.com/3155568/174642402-2810e281-5470-4a78-ad65-1f8c544d1f37.png)

## Questions

1. How do I make it work?
2. I understand why there could be a need to define a global project, but why is it a requirement?

**Comment by apupier** [comment]
Given that Vale Server is deprecated, it is planned to be removed from this extension https://github.com/errata-ai/vale-vscode/issues/71

**Comment by ChrisChinchilla** [comment]
This might be closed by #78 @politician let me know if the latest extension release doesn't fix things

**Comment by ChrisChinchilla** [comment]
Fixed by #108
